src/main/java/com/mycelium/spvmodule/guava/Bip44NotificationManager.kt

```diff
@@ -50,44 +50,26 @@ class Bip44NotificationManager {
             }
         }
 
-        notification = if (peerCount == 0 || blockchainState == null) {
-            buildNoPeerNotification()
-        } else {
-            buildSuccessfulConnectionNotification()
-        }
+        notification = buildSuccessfulConnectionNotification()
```

Comment:
should it really be `buildSuccessfulConnectionNotification` if it covers both cases?